README.md

```diff
@@ -1,5 +1,20 @@
-# mudlet-scripts
-Official Mudlet scripts
+# Warlock MUD - skrypty dla Mudleta
 
+To repozytorium zawiera paczkę podstawowych skryptów przeznaczonych dla polskiego MUDa Warlock. Są one automatycznie 
+ściągane przez Mudleta w momencie połączenia się z Warlockiem. 
 
-https://github.com/Talisterus/warlock-script
+
+## Dane podstawowe
+
+* Strona gry: http://warlock.pl/
+* GitHub Mudlet: https://github.com/Mudlet/Mudlet
+* Installer Mudlet: https://github.com/Mudlet/installers
+* Pierwotne źródło skryptów: https://github.com/Talisterus/warlock-script
```

Comment:
To chyba nie prawda..
To bylo napisane przez Garagotha od 0 o ile wiem..

Reply:
A to usunę - bo było w poprzedniej wersji README to nie chciałem by ktoś się poczuł urażony